examples/passport-oauth2-login/src/application.ts

```diff
@@ -0,0 +1,48 @@
+// Copyright IBM Corp. 2020. All Rights Reserved.
+// Node module: @loopback/example-passport-oauth2-login
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {BootMixin} from '@loopback/boot';
+import {RepositoryMixin} from '@loopback/repository';
+import {RestApplication} from '@loopback/rest';
+import {ServiceMixin} from '@loopback/service-proxy';
+import {MySequence} from './sequence';
+import {AuthenticationComponent} from '@loopback/authentication';
+import {Oauth2Controller} from './controllers';
```

Comment:
Should we use `OAuth2` instead of `Oauth2`?

Reply:
agreed